asreviewcontrib/dory/feature_extractors/transformer_embeddings.py

```diff
@@ -94,20 +147,23 @@ def fit(self, X, y=None):
         # for sentence-transformers, so return self
         return self
 
-    def fit_transform(self, X, y=None):
+    def transform(self, X, y=None):
         if self.verbose:
             print("Embedding text...")
 
-        embeddings = self._model.encode(
-            X, show_progress_bar=self.verbose, normalize_embeddings=self.normalize
```

Comment:
I wonder what the `normalize_embedding` does when set to `True`. Just curiosity.

Reply:
It normalizes using torch nn: `embeddings = torch.nn.functional.normalize(embeddings, p=2, dim=1)`

Reply:
Oke, that's L2.